### ISSUE TITLE
trying to change the loss

### ISSUE THREAD

**Issue by JaspervDalen** [experiencor/keras-yolo3#69]
Hi @experiencor 

I am trying to train a network on a dataset where each image is labeled with 3 annotations (all correct) and I want to predict 1 boundingbox. To do this I need to change the loss in such a way that the prediction is only compared with the annotation bounding box that has the highest iou of all 3. However when I look at your loss function i am not sure what I need to change to do this...

Can you give me some help with this or do you not have the time?

kind regards,

Jasper

**Comment by JaspervDalen** [comment]
Is it correct if I say that this is already so in the current loss? It does not seem to do anything with false negatives? Or did I miss something?
